packages/core/strapi/lib/commands/actions/ts/generate-types/command.js

```diff
@@ -10,12 +10,7 @@ module.exports = ({ command }) => {
   command
     .command('ts:generate-types')
     .description(`Generate TypeScript typings for your schemas`)
-    .option(
-      '-o, --out-dir <outDir>',
-      'Specify a relative directory in which the schemas definitions will be generated'
-    )
-    .option('-f, --file <file>', 'Specify a filename to store the schemas definitions')
-    .option('--verbose', `Display more information about the types generation`, false)
+    .option('-d, --debug', `Run the generation with debug messages`, false)
```

Comment:
Would it make sense to have a vebosity that could be set to info, debug, error, silent, etc?